docs/user-guide/team-planner/README.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_navigation:
+  title: Team planner
+  priority: 890
+description: Using the team planner in OpenProject.
+robots: index, follow
+keywords: team planner,planning calendar, resource management
+---
+
+# Team planner
+
+The team planner is a module that can assist getting a better overview of which team member is working on what work package at what time. It is also a useful tool to schedule those team members for different tasks on various times.
+
+A few notes: 
+
+1. The work package module has to be enabled as a prerequisite for the team planner
+2. You can have multiple public, private, as well as favorite team planning calendars. They will all appear in the left menu bar.
+
+| Topic                 | Content                                     |
+| ------------------------------------- | :------------------------------------------ |
+| [Main view - planning calendar basics](team-planner) | How to navigate the team planner? |
+| **Getting started** |  |
+| [Step 1: Adding team members](team-planner) | How to add team members? |
+| [Step 2: Adding/working with work package strips](team-planner) | How to work with work package strips? |
+| [Work package strip detail view](team-planner) | How to access the work package detail view? |
+| [Removing a work package strip](team-planner) | How to remove a work package strip? |
```

Comment:
The anchor links in this table are missing. e.g. `[Removing a work package strip](#removing-a-work-package-strip)`